packages/phrases/src/locales/en/translation/admin-console/mfa.ts

```diff
@@ -15,14 +15,19 @@ const mfa = {
   webauthn_domain_tip:
     'WebAuthn binds public keys to the specific domain. Modifying your service domain will block users from authenticating via existing passkeys.',
   backup_code: 'Backup code',
-  backup_code_description: 'Generate 10 unique codes, each usable for a single authentication.',
-  backup_code_setup_hint: 'The backup authentication factor which can not be enabled alone:',
+  backup_code_description: 'Generate 10 one-time backup codes after users set up any MFA method.',
+  backup_code_setup_hint: "When users can't verify the above MFA factors, use the backup option.",
   backup_code_error_hint:
-    'To use Backup code for MFA, other factors must be turned on to ensure your users successful sign-in.',
+    'To use a backup code, you need at least one more MFA method for successful user authentication.',
   policy: 'Policy',
-  two_step_sign_in_policy: '2-step authentication policy at sign-in',
-  user_controlled: 'Users have the choice to enable MFA personally.',
-  mandatory: 'Mandatory MFA for all your users at every sign-in.',
+  policy_description: 'Set the MFA policy for sign-in and sign-up flows.',
+  two_step_sign_in_policy: '2-step verification policy at sign-in',
+  user_controlled: 'Users can enable or disable MFA on their own',
+  user_controlled_tip:
+    'Users can skip the MFA set-up the first time at sign-in or sign-up, or enable/disable it in account settings.',
+  mandatory: 'Users are always required to use MFA at sign-in',
+  mandatory_tip:
+    'Users must set up MFA the first time at sign-in or sign-up, and use it for all future sign-ins.',
```

Comment:
These phrases also need to be updated.

<img width="446" alt="image" src="https://github.com/logto-io/logto/assets/12833674/e3847a3a-47ba-42d6-9187-a651bf8c5166">

Reply:
Your pic is out-dated.